README.adoc

```diff
@@ -43,18 +43,19 @@ database operations and takes care of database vendor-specific differences. This
 focus on the business logic of your application instead of wasting time implementing repetitive CRUD logic.
 
 The application that you will be working with is an event manager, which is composed of a UI
-that uses MicroProfile Rest Client to access RESTful services and an event microservice 
-for creating, retrieving, updating, and deleting events. In this guide, you will be focused 
-on the event microservice. The database logic is implemented in a Data Access Object (DAO) 
-to isolate the database operations from the rest of the application. This DAO accesses 
-and persists data model instances to the database and can be injected and consumed by other 
-components in the application. An Embedded Derby database is used as a data store for all the events.
+and an event microservice for creating, retrieving, updating, and deleting events. In this 
+guide, you will be focused on the event microservice. The event microservice consists of
+a JPA entity class whose fields will be persisted to a database. The database logic is implemented in 
+a Data Access Object (DAO) to isolate the database operations from the rest of the service. 
+This DAO accesses and persists JPA entities to the database and can be injected 
+and consumed by other components in the microservice. An Embedded Derby database is used 
+as a data store for all the events.
 
 You will use JPA annotations to define an entity class whose fields are persisted to the 
-database. The interaction between your application and the database is mediated by the persistence 
+database. The interaction between your service and the database is mediated by the persistence 
 context that is managed by an entity manager. In a Java EE environment, one type of entity
-manager is a container-managed entity manager. In this guide you will use dependency injection
-to acquire a container-managed entity manager, placing the burden of opening and closing 
+manager is a container-managed entity manager. In this guide you will use a container-managed
```

Comment:
This makes me want to leave the guide and Google "types of java ee entity managers".

Reply:
It also makes me wonder "what exactly is an entity manager and why is it necessary?"